app/views/browse/deployment-config.html

```diff
@@ -47,12 +47,12 @@
                       <a ng-href="project/{{projectName}}/set-limits?dcName={{deploymentConfig.metadata.name}}"
                         role="button">Set Resource Limits</a>
                     </li>
-                    <li ng-if="!autoscalers.length && ('extensions/horizontalpodautoscalers' | canI : 'create')">
+                    <li ng-if="!autoscalers.length && ({resource: 'horizontalpodautoscalers', group: 'autoscaling'} | canI : 'create')">
```

Comment:
The JS code makes requests for `horizontalpodautoscalers` in the `extensions` group. @liggitt could the mismatch cause a problem?